var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -751,12 +751,13 @@ def find_optional_library(name, prefix):
                     )
                     gfortran_lib = LibraryList(libfile)
                     sp_lib += [ld_flags_from_library_list(gfortran_lib)]
+                mpi = strumpack["mpi"]
```

Comment:
This will need to check `if "+mpi" in strumpack` before grabbing `strumpack["mpi"]

You would need to indent the following if/else statement inside of that check as well.

Reply:
Gah yes... fixing now.